pom.xml

```diff
@@ -451,7 +445,7 @@
         <dependency>
             <groupId>org.bouncycastle</groupId>
             <artifactId>bcprov-jdk18on</artifactId>
-            <version>1.78.1</version>
+            <version>1.78</version>
```

Comment:
Is this not a downgrade? `1.78.1` was released after `1.78` and seems to [fix some issues](https://www.bouncycastle.org/resources/java-release-1-78-1-is-now-available-for-download/).